controllers/cloud.redhat.com/providers/iqe/impl.go

```diff
@@ -41,10 +40,12 @@ func CreateIqeJobResource(cache *providers.ObjectCache, cji *crd.ClowdJobInvocat
 	}
 
 	envvar := []core.EnvVar{
-		{Name: "ACG_CONFIG", Value: "/cdapp/cdappconfig.json"},
 		{Name: "ENV_FOR_DYNACONF", Value: cji.Spec.Testing.Iqe.DynaconfEnvName},
 		{Name: "NAMESPACE", Value: nn.Namespace},
-		{Name: "CLOWDER_ENABLED", Value: "true"},
+		{Name: "IQE_DEBUG_POD", Value: strconv.FormatBool(cji.Spec.Testing.Iqe.Debug)},
```

Comment:
To avoid some confusion, this line controls how the debug pod starts: https://gitlab.cee.redhat.com/insights-qe/iqe-dockerfiles/-/merge_requests/59/diffs#95dd46e5eb3ea3c869a1eec9b1938e0649aadc52_13_14